crates/sui-types/src/effects/effects_v2.rs

```diff
@@ -411,7 +412,7 @@ impl TransactionEffectsV2 {
         executed_epoch: EpochId,
         gas_used: GasCostSummary,
         shared_objects: Vec<SharedInput>,
-        loaded_per_epoch_config_objects: BTreeSet<ObjectID>,
+        loaded_per_epoch_config_objects: BTreeMap<ObjectID, Option<SequenceNumber>>,
```

Comment:
```suggestion
        unsequenced_per_epoch_config_objects: BTreeMap<ObjectID, Option<SequenceNumber>>,
```
I know this was the existing name, but we should really highlight the access pattern here